Update README.md (#12909)
DataDog/integrations-core: helm/README.md

This is a cluster check. For more information, see the [Cluster checks documentation][2].

> **Note**: It is important to note that an empty confd can be passed if there is no configuration required

Reviewer: ```suggestion
**Note**: If no configuration is required, an empty `conf.d` can be passed.
```
